CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog of FOSSology
 
+### 4.1.0 (May 12th 2022)
+
+This release adds important corrections to
+[4.0.0](https://github.com/fossology/fossology/releases/tag/4.0.0)
+
+The release 4.1.0 introduces new agent `ScanCode`, used to scan for licenses,
+copyrights etc.
+
+The release 4.1.0 also introduces new feature to automatically deactivate the
+copyrights and cutter removal. There is a special note about this feature.
+> As this feature can still be improved, we are marking this as `experimental`
+and not recomended for productive instances.
+> Also this feature requires to install additional dependencies. One needs to
+run fo-postinstall with --python-experimental.
+
+The release 4.1.0 introduces a number of corrections to
+[4.0.0](https://github.com/fossology/fossology/releases/tag/4.0.0)
+and major changes to FOSSology, including:
+
+* Security fix for JWT tokens
+* Migration fix for copyrights
+
+#### Credits to contributors for 4.1.0
+
+From the GIT commit history, we have following contributors since
+[4.0.0](https://github.com/fossology/fossology/releases/tag/4.0.0):
```

Comment:
From the GIT commit history, we have following contributors since
 [4.1.0-rc1](https://github.com/fossology/fossology/releases/tag/4.1.0-rc1)

Reply:
The list is from 4.0.0. From RC1, we have only 2 contributors.